libcudacxx/include/cuda/latch

```diff
@@ -11,6 +11,17 @@
 #ifndef _CUDA_LATCH
 #define _CUDA_LATCH
 
+#include <cuda/std/detail/__config>
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+#include <cuda/std/__latch/latch.h>
```

Comment:
Should this not be `<cuda/__latch/latch.h>`?

Reply:
It sure should
```suggestion
#include <cuda/__latch/latch.h>
```